test/functional/services/apps_menu.ts

```diff
@@ -59,6 +59,7 @@ export class AppsMenuService extends FtrService {
     if (!(await this.testSubjects.exists('collapsibleNav'))) {
       await this.testSubjects.click('toggleNavButton');
     }
+    await this.testSubjects.exists('collapsibleNav');
```

Comment:
This helper wasn't confirming a 'happy-path', we should assert that the nav flyout does appear before progressing to use any of the nav methods (clicking an app title for example)